src/cluster.rs

```diff
@@ -134,14 +135,31 @@ impl ClusterConnection {
         true
     }
 
+    /// Get a new ClusterPipeline
+    pub fn pipe(&self) -> ClusterPipeline {
```

Comment:
If I remember way back when I originally implemented this, I think it was to prevent someone from accidentally calling `pipe()` when they needed a `ClusterPipeline`. This way, the only way to get a `ClusterPipeline` was through a `ClusterConnection`. But there are already other catches to prevent execution of a `Pipeline` with a `ClusterConnection`, and vice-versa. I'll change the PR to add a `cluster_pipe` constructor instead :+1: